### ISSUE TITLE
Some images not loading properly on Kusama

### ISSUE THREAD

**Issue by prury** [kodadot/nft-gallery#7596]
### What happened?

https://canary.kodadot.xyz/rmrk/explore/items?listed=true&min=100000000000000

![image](https://github.com/kodadot/nft-gallery/assets/36627808/8f918644-edb4-46c8-a606-9befdfc23226)

**Comment by kodabot** [comment]
ASSIGNED - @Jarsen136 🔒 LOCKED -> Thursday, October 12th 2023, 21:05:30 UTC -> 36 hours

**Comment by yangwao** [comment]
I sense some wrong merging ?

**Comment by roiLeo** [comment]
> I sense some wrong merging ?

nah same on prod this is an old issue

**Comment by yangwao** [comment]
> nah same on prod this is an old issue

i see https://kodadot.xyz/rmrk/explore/items?listed=true&min=100000000000000

**Comment by Jarsen136** [comment]
> > nah same on prod this is an old issue
> 
> i see https://kodadot.xyz/rmrk/explore/items?listed=true&min=100000000000000

The issue has been fixed in the production environment now.
<img width="1826" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/a7cde82e-501b-493a-b506-58e86f89cfae">


I found the root reason is that the ipfs gateway side does not return the correct 'content-type' for the metadata of nft. After I force fetching the original metadata with PR #7620, the 'content-type' would change to the correct one, and then it load properly.
